backend-db/Rhyolite/Backend/DB/PsqlSimple/Orphans.hs

```diff
@@ -2,142 +2,9 @@
 module Rhyolite.Backend.DB.PsqlSimple.Orphans where
 
 import Database.PostgreSQL.Simple
-import Database.PostgreSQL.Simple.FromField
-import Database.PostgreSQL.Simple.FromRow
 import Database.PostgreSQL.Simple.ToField
 import Database.PostgreSQL.Simple.ToRow
 
-instance (FromField a, FromField b, FromField c, FromField d, FromField e,
```

Comment:
can you include a note in the changelog about where these instances can now be found?

Reply:
I thought of that but since they're now upstreamed it's literally impossible for you to *not* have them. I.e. there's nothing anyone can do in response to this change other than possibly delete imports of this file?

Reply:
I guess that might be enough of a reason. I'll add it.